compiler/surface/lexer_en.cppo.ml

```diff
@@ -111,6 +111,9 @@
 (* Directives *)
 
 #define MR_LAW_INCLUDE "Include"
+#define MR_MODULE_DEF "Module"
```

Comment:
Could you add these to all the syntax highlighters ? I know it's a pain but it's useful...

Reply:
It's in the tree-sitter syntax already :upside_down_face: 

Joke aside, I think we'll want to add a specific declaration for externally-implemented modules (maybe just `> Module Foo external` ?) so might be a good idea to settle on this and do all of them in one go

Reply:
Hm I had a look but I can't figure if there is any support for directives at the moment... I didn't find any reference to the existing `Include` for example.

Reply:
`> module Foo external` is a good idea!